### ISSUE TITLE
EA not showing in MT5

### ISSUE THREAD

**Issue by Feli555** [darwinex/dwx-zeromq-connector#6]
Im putting the file in the experts folder, but it does not show up, and when I go to compile it it shows 100 errors when I change it to MQL5
![image](https://user-images.githubusercontent.com/14276622/56324041-59d70180-6133-11e9-9fdc-a655e5ff31a8.png)

**Comment by integracore2** [comment]
Hi @Feli555 ,

This project supports MetaTrader 4 at the present time, not MetaTrader 5.

An MQL4 EA cannot be deployed directly to MT5 due to certain language and platform differences - hence the errors you’re getting.

Stay tuned - we are already working on a version for MQL5 and will update the GitHub repository when it’s complete and tested.

Many thanks for your interest!
